docs/topics_5/review-reports.adoc

```diff
@@ -276,12 +276,12 @@ The dependency graph may be adjusted by using any of the following.
 * Clicking on any item in the legend will enable or disable all items of the selected type. For instance, selecting the embedded WARs icon will disable all embedded WARs if these are enabled, and will enable these dependencies if they are disabled.
 
 [id='shared_archives_{context}']
-== Archives Shared by Multiple Applications
+== Shared Libraries: Archives shared by multiple applications

```

Comment:
Hard item to caption properly. 
               
![image](https://user-images.githubusercontent.com/60698649/86933839-af863800-c143-11ea-8699-c34270f107bb.png)

Reply:
If you are documenting a concept, you use sentence case. If you are documenting the UI itself, you document the UI.

This is a concept. You do not have to slavishly follow what the UI says. In fact, in many cases, the UI might be too terse or ambiguous to understand easily, so you would have to explain the UI clearly (and then submit UI bug to fix the English). So use sentence case. Doesn't matter what the UI says. You're documenting a concept here.